storage/postgres/tree_storage.go

```diff
@@ -141,6 +145,13 @@ func (p *pgTreeStorage) getStmt(ctx context.Context, skeleton *statementSkeleton
 	}
 
 	statement, err := expandPlaceholderSQL(skeleton)
+
+	var counter = skeleton.restPlaceholders*skeleton.num + 1
```

Comment:
golang: This can be simply `counter := skeleton... + 1`

Reply:
That's good to know; I'll update.